src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,419 @@
+* **Task context:** "Working on ticket #123", "In delivery zone A".
+* **User info:** Use presence to show an avatar, display name, role, or other information about the user.
+
+This contextual presence information helps participants understand not just who's online, but who's ready and able to engage with the current task.
+
+```javascript
+// subscribe to presence events
+const { unsubscribe } = room.presence.subscribe((event) => {
+  console.log(`${event.member.clientId} entered with data: ${event.member.data}`);
+
+  const presentMembers = await room.presence.get();
+  console.log('Present members:', presentMembers);
+});
+
+// join presence
+await room.presence.enter({ status: 'available' });
+
+// change presence data
+await room.presence.enter({ status: 'busy' });
+
+// leave presence
+await room.presence.leave();
+```
+
+## Typing indicators
+
+[Typing indicators](/docs/chat/rooms/typing) are a common feature in most chat applications. They show when someone is actively composing a message, helping to:
+
+* **Manage expectations:** Users know when a response is being prepared.
+* **Reduce duplicate messages:** See that someone is already addressing the question.
+* **Improve flow:** Better conversation pacing in support and collaborative scenarios.
+
+In Ably Chat typing indicators are a core feature with a simple API:
+
+```javascript
+const room = await chatClient.rooms.get('support-ticket-123');
+
+// show who's typing
+room.typing.subscribe((event) => {
+  console.log('Currently typing:', event.currentlyTyping);
+});
+
+// show a typing indicator (call on every keystroke, Ably Chat SDKs throttle this to a predefined interval)
+await room.typing.keystroke();
+
+// stop the typing indicator (for example when a message is sent)
+await room.typing.stop();
+```
+
+## Message reactions
+
+Message reactions are a great way to enhance engagement in a task-oriented chat and enable users to quickly express sentiment to key points in the conversation.
+
+**Message reactions** provide granular feedback on specific content:
+
+* **Validate information:** ✅ for confirmed details in deliveries or transactions.
+* **Request clarification:** ❓ for questions about specific messages.
+* **Show appreciation:** ⭐ for helpful responses in support.
+
+Send a message reaction:
+<Code>
+```javascript
+const message; // your message
+await room.messages.reactions.send(message.serial, {name: '✅'});
+```
+</Code>
+
+Message reactions in Ably Chat come in three types: `unique`, `distinct` and `multiple`, to suit different use cases: from one reaction per message to multiple reactions per message with or without counts. See the [Message reactions](/docs/chat/rooms/message-reactions) documentation for more details.
+
+## Message history: essential task context <a id="history"/>
+
+Message history is crucial for task-oriented chats, ensuring continuity and context even when participants join mid-task or return after interruptions.
```

Comment:
Worth a link somewhere in here to the BYOD guide (when it arrives) as I think audit is key for many of these use-cases